site/content/blog/lowrisc-6th-birthday-openuk-award/index.md

```diff
@@ -0,0 +1,21 @@
++++
+date = "2020-10-22T12:00:00+00:00"
+title = "A birthday present for lowRISC: We won an OpenUK Award!"
+featured_image = "lowrisc-openuk-award.jpg"
+
+[[resources]]
+src = "lowrisc-openuk-award.jpg"
+title = "lowRISC's 2020 OpenUK Award"
++++
+
+{{< img class="float-right rounded m-2" width=300 src="lowrisc-openuk-award.jpg" title="lowRISC's 2020 OpenUK Award" >}}
+
+On October 20th lowRISC CIC won in the Open Hardware category at the [2020 OpenUK Awards](https://openuk.uk/awards/), describing lowRISC as “the jewel in the crown of the UK’s open silicon companies”. The OpenUK awards promote “UK Leadership in Open Technology”, and are given out by OpenUK, a UK-based not-for-profit company which supports open source collaboration and open technologies within the United Kingdom.
+
+On receiving the award lowRISC CTO, Alex Bradbury, said “We're incredibly grateful to have been recognised for our achievements and contributions to date. Six years into our journey, it's a fantastic time to reflect on and thank the many people helping to make our vision for open source silicon a reality - our community of contributors, project partners, and the growing lowRISC team.”
+
+What a great present to receive on the occasion of lowRISC’s 6th Birthday! It doesn’t seem too long ago we were celebrating our 5th birthday but what a year it has been. The pioneering Open Silicon project [OpenTitan](https://www.opentitan.org) had its public launch, we’ve moved to a great new office, hired new staff and are making fantastic progress on OpenTitan, Ibex and LLVM.
+
+We’ve had our heads down working hard so haven’t been writing about our work as much as we’d like but we’ll be getting a larger update out soon. In the meantime you can always see what we’re up to on our [GitHub Repositories](https://www.github.com/lowRISC) and contribute yourself.
```

Comment:
This would feel less breathless with some commas. I'd suggest one after "hard" and one after "meantime".

Also, I'm not sure that anyone wants to contribute themselves to the cause :-) Maybe "get involved yourself" ?